alacritty/res/text.v.glsl

```diff
@@ -1,18 +1,24 @@
 #version 330 core
 // Cell properties.
-layout (location = 0) in vec2 gridCoords;
+layout(location = 0) in vec2 gridCoords;
 
 // Glyph properties.
-layout (location = 1) in vec4 glyph;
+layout(location = 1) in vec4 glyph;
 
 // uv mapping.
-layout (location = 2) in vec4 uv;
+layout(location = 2) in vec4 uv;
 
-// Text foreground rgb packed together with multicolor flag.
-layout (location = 3) in vec4 textColor;
+// Text foreground rgb packed together with cell flags. textColor.a
+// is a bitflag with the following values.
+//
+// 0 - empty
+// 1 - wide char
+// 2 - colored glyph
+//
```

Comment:
```suggestion
// Text foreground rgb packed together with cell flags. textColor.a
// is a bitflag with the following values.
//
// 0 - empty
// 1 - wide char
// 2 - colored glyph
```

We usually don't add spaces after our comments.

Also I'm not sure if we want to document the actual *values* of the cell flags, that seems a bit strange to me for bitflags. Instead we should document the individual bits? Or how would this list continue, I'd assume with `4 - whatever`?